File: Python/guessingNum.py
import random

diff = input("Enter number of zeros: ")
diff = int(diff)

num = random.randint(0, 10**diff + 1)

print ("Generated number from 0-" + str(10**diff))

guess = -1
guessCount = 0

while guess != num:
    guess = input ("Enter guess: ")
    guess = int(guess)

    guessCount += 1
    
    if guess > num:
        print("Too high")
    elif guess < num:
        print("Too low")

print("Congradulations!")
print("Guesses:", guessCount)

File: JavaScript/guessingNum.js
const read = require('readline-sync')

let guess = -1
let guessCount = 0

let diff = read.question("Enter number of zeros: ")

let num = Math.floor(Math.random() * Math.pow(10, diff))

console.log("Generated number from 0-" + Math.pow(10, diff))

while (guess != num)
{
	guess = read.question("Enter guess: ")

	guessCount++

	if (guess > num)
		console.log("Too high")
	else if (guess < num)
		console.log("Too low")
}

console.log("Congradulations!")
console.log("Guesses: " + guessCount)
